CHANGELOG.md

```diff
@@ -1,9 +1,17 @@
 # v1.40.0
 
+#### Default for enable windows changed
+
+The default value for enableWindows has been changed from false to true. This change makes it so the node damemonset will be scheduled on windows nodes by default. If you are a addon customer you can ignore this message as this has always been the default addon behavior. If you wish to not have the node daemonset scheduled on your windows nodes you will need to change enableWindows to false.
```

Comment:
This should be in the Helm changelog, as it's a helm-only change.

Reply:
Also, remove everything after "If you are a addon customer", nothing related to the addon should be documented here.